src/state/serviceInfo/actions.js

```diff
@@ -3,11 +3,15 @@
 import { SERVICE_INFO_RECEIVE, USER_RECEIVE } from 'console/state/action-types';
 import { makeApiRequest } from 'console/state/requests/actions';
 
-export function fetchServiceInfo() {
+export function fetchServiceInfo(accessToken) {
   return async dispatch => {
     const requestId = 'fetch-service-info';
     const serviceInfo = await dispatch(
-      makeApiRequest(requestId, 'v2/service_info/'),
+      makeApiRequest(requestId, 'v2/service_info/', {
+        headers: {
+          Authorization: `Bearer ${accessToken}`,
```

Comment:
This might be best to abstract so you don't have to get the access token as a prop or a function argument every time but to read from the store or something.